releases/in-progress.html

```diff
@@ -16,10 +16,12 @@
       <h5>Improvements to the console log</h5>
 
       <p>
-        The console log now supports rendering of ansi color codes to show you much better logs.
+        Not only does go now support rendering of ansi color codes to show you much beautiful console logs,
+        we use much more screen real-estate to show you what you need to see. Additionally, go will automatically
```

Comment:
How about: "much more real-estate is used to show you what you need to see. Additionally, Go will ..."?